qiskit/aqua/components/neural_networks/quantum_generator.py

```diff
@@ -308,6 +316,46 @@ def objective_function(params):
 
         return objective_function
 
+    def _convert_to_gradient_function(self, gradient_object, quantum_instance, discriminator):
+        """
+        Convert to gradient function
+
+        Args:
+            gradient_object (Gradient): the gradient object to be used to
+                compute analytical gradients.
+            quantum_instance (QuantumInstance): used to run the quantum circuit.
+            discriminator (torch.nn.Module): discriminator network to compute the sample labels.
+
+        Returns:
+            gradient_function: gradient function that takes the current
+                parameter values and returns partial derivatives of the loss
+                function w.r.t. the variational parameters.
+        """
+        def gradient_function(current_point):
+            """
+            Gradient function
+
+            Args:
+                current_point (np.ndarray): Current values for the variational parameters.
+
+            Returns:
+                loss_gradients (np.ndarray): array of partial derivatives of the loss
```

Comment:
> These couple lines of code seem to be the source of the linting error that the checks threw - I simply followed the format used with the function _get_objective_function - hopefully that's okay?

You need to fix this line.

```suggestion
                np.ndarray: array of partial derivatives of the loss
```